mailpoet/assets/js/src/automation/listing/index.tsx

```diff
@@ -143,21 +147,15 @@ export function AutomationListing({ workflows, loading }: Props): JSX.Element {
               allowFreeTextSearch
```

Comment:
Not sure if it's related to this PR, but our search does currently not work. This happens, when I type something in the search box: 
![image](https://user-images.githubusercontent.com/6458412/196598274-c17afaac-b7c0-4a15-93eb-e0d195d7a519.png)

Reply:
Yeah, we never implemented it. I'm thinking maybe we should hide it for the MVP 🤔